.github/actions/openvino_provider/action.yml

```diff
@@ -0,0 +1,156 @@
+name: 'OpenVINO provider'
+description: 'Provides OpenVINO build artifacts from a requested source'
+inputs:
+  revision:
+    description: 'Version of OpenVINO to provide - latest_available | HEAD | specific commit hash'
+    required: true
+  branch_name:
+    description: 'Branch of OpenVINO to take the revision from if no specific hash was provided. 
+                  Taken from github context by default'
+    required: false
+  ov_artifact_name:
+    description: "Name under which to upload provided OpenVINO build artifacts, set automatically by default"
+    required: false
+  commit_packages_to_provide:
+    description: "Comma-separated list of OpenVINO packages from post-commit to provide, all available by default"
+    required: false
+  commit_build_key:
+    description: 'public_linux_ubuntu_20_04_release | public_linux_ubuntu_22_04_release | 
+                  public_linux_ubuntu_24_04_release | public_windows_vs2019_release'
+    required: false
+  commit_share_path:
+    description: 'Path to share holding OpenVINO post-commit artifacts'
+    required: false
+    default: '/mount/build-artifacts'
+  nightly_build_key:
```

Comment:
hope we can align nightly and commit build keys to be just "platform"

Reply:
We thought about this. We cannot directly align platform names in nightly packages with post-commit (since the latter is required for compatibility with internal validation; and changing the former is problematic (discussed with @artanokhov), so we would need to maintain the map to provide a unified platform key here, so for now we decided that it's easier to keep these keys separate

Reply:
right, maybe we can tackle this problem in future :)